flashcards/docker-compose.yaml

```diff
@@ -32,6 +32,7 @@ services:
       - "80:80"
     volumes:
       - ./backend/staticfiles:/app/staticfiles
+      - ./backend/media:/app/media
```

Comment:
I would also add media to prod file

Reply:
same like here or with media_volume?

Reply:
please follow the quiz [prod](https://github.com/AzenoIT/azeno/blob/master/quiz/docker-compose.prod.yaml) file. The way it is done in dev version is not suited for production at all.

Reply:
in quiz prod file media is not set, 
that's why am asking whether new media volume is required?

Reply:
you can check the badge model pull request, or do it similarily to staticfiles

Reply:
done